app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self, comfort_class: int, clean_mark: int, brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = 'BMW'
```

Comment:
The assignment of the brand in the Car class constructor is hardcoded to 'BMW'. This should be set to the 'brand' parameter passed to the constructor to allow for different car brands.